src/pseudopeople/configuration/validator.py

```diff
@@ -15,6 +15,13 @@
 from pseudopeople.schema_entities import DatasetSchema
 
 
+class ParameterConfigValidator(Protocol):
+    def __call__(
```

Comment:
Oh my. What was the error message that led to this? Why couldn't you just use Callable?

Reply:
I don't remember exactly tbh. I could try it again without the Protocol and see if it still fails.

Reply:
I guess it doesn't really matter. Was just surprised to see a "new" typing class (Protocol) and neeing to define __call__ just to get out of a mypy error